### ISSUE TITLE
Move Scheduler related spi util classes into schedulers package

### ISSUE THREAD

**Issue by billonahill** [twitter/heron#1604]
The following classes only seem to be required for schedulers and do not need to be in spi. Instead they could probably live in `c.t.h.scheduler.utils`:
```
c.t.h.spi.utils.Runtime
c.t.h.spi.utils.SchedulerConfig
c.t.h.spi.utils.Shutdown
c.t.h.spi.utils.LauncherUtils
```

`SchedulerConfig` should also not be a util.

**Comment by mycFelix** [comment]
@billonahill - I send PR for this. Please review when you have a chance

**Comment by mycFelix** [comment]
merged and closed

**Comment by objmagic** [comment]
resolved via #1617
